src/App.tsx

```diff
@@ -17,8 +21,7 @@ const App: React.FC<{}> = () => {
     <div className="App">
       <header className="header"></header>
       <div className="container">
-        <CreatePost addPost={postService.addPost} categoryService={categoryService.result} />
-        <h1>Epic Posts </h1>
+        <CreatePost reFetchPosts={reFetchPost} categoryService={categoryService.result} />
```

Comment:
You can alternatively pass `postService.fetchPosts` directly like `reFetchPosts={postService.fetchPosts}` So then you don't need extra wrapper function on line 11 👍